tests/conftest.py

```
@@ -0,0 +1,46 @@
+from pathlib import Path
+
+import pytest
+from icalendar import Calendar
+
+from tests.factories import CalendarFactory
+
+FIXTURES_DIR = Path(__file__).parent / "fixtures" / "ics"
+
+
+@pytest.fixture()
+def calendar(db):
```

Ruff (0.15.2)

[warning] Unused function argument: `db`

(ARG001)